template/profile.html

```diff
@@ -0,0 +1,130 @@
+      height: 200px;
+      display: inline-block;
+      margin-left: 10px;
+      margin-right: 10px;
+      transition: all .2s ease-in-out;
+    }
+
+    .user-game.mdl-card:hover { transform: scale(1.1); }
+
+    .user-game img {
+      width: 200px;
+      height: 150px;
+      background-color: lightgray;
+    }
+
+    .user-game-name{
+      margin-left: 10px;
+      margin-top: 10px;
+    }
+  </style>
+
+  <script>
+    Polymer({
+      is: "user-game",
+    })
+  </script>
+</dom-module>
+
+<dom-module id="user-achievement">
+  <template>
+    <mdl-chip class="user-achievement">
+      <mdl-chip-contact class="mdl-color--amber mdl-color-text--white user-achievement-image"></mdl-chip-contact>
+      <mdl-chip-text class="user-achievement-name"></mdl-chip-text>
+    </mdl-chip>
+  </template>
+  <style>
+  </style>
+
+  <script>
+    Polymer({
+      is: "user-achievement",
+    })
+  </script>
+</dom-module>
+
+<dom-module id="papan-user-profile" class="user-page">
+  <template>
+    <mdl-card class="user-info">
+      <img src="" id="picture"/>
+      <mdl-card-title-tex id="nickname">
+        Nickname
+      </mdl-card-title-tex>
+    </mdl-card>
+
+    <div id="user-games-list">
+      <div class="category">Games</div>
+    </div>
+
+    <div id="user-achievements-list">
+      <div class = "category">Achievements</div>
+    </div>
+  </template>
+
+  <style>
+    .user-info {
+      width: auto;
+      height: 250px;
+    }
+
+    .user-info #nickname{
+      margin-top: 10px;
+      margin-left: 10px;
+      font-size: 20pt;
+    }
+
+    .user-info img {
+      margin-top: 10px;
+      margin-left: 10px;
+      width: 200px;
+      height: 200px;
+      background-color: lightgray;
+    }
+
+    .category {
+      font-size: 20pt;
+      border-bottom: 1px solid lightgrey;
+      margin-top: 10px;
+      margin-bottom: 10px;
+      padding-bottom: 5px;
+    }
+  </style>
+
+  <script>
+    Polymer({
+      is: "papan-user-profile",
+
+      ready: function() {
+        //populate list of games
+        var list = document.getElementById('user-games-list');
+        var games = ["Chess", "Tic tac toe", "Falken's Maze", "Poker"];
+        for (i = 0; i < games.length; i++) {
+          var game = document.createElement('user-game');
+          game.getElementsByClassName('user-game-name')[0].innerHTML = games[i];
+          list.appendChild(game);
+        }
+
+        //populate list of achievements
+        var list = document.getElementById('user-achievements-list');
+        var achievements = ["Gamer", "Master of nuggets", "Killed em' all"];
+        for (i = 0; i < achievements.length; i++) {
+          var achievement = document.createElement('user-achievement');
+          achievement.getElementsByClassName('user-achievement-name')[0].innerHTML = achievements[i];
+          achievement.getElementsByClassName('user-achievement-image')[0].innerHTML = achievements[i][0];
+          list.appendChild(achievement);
```

Comment:
It would be better to use Polymer's API to access/change the DOM:
- for access: in fact, we need only to scan the current component, which is seen by Polymer (shadow/shady DOM), so it is possible to use $ and $$ as selectors (like you did this.$.gameHeader to get the game header. A syntax that differs only a little (query selectors) also works for classes).
- for change: for now (Polymer 1.*), it is better to use Polymer's API to change the DOM so that the shadow/shady DOM trees are updated properly by Polymer. Otherwise, there might be some surprises with the rendering, and handling of the DOM tree.